extension/src/project/readOnlyDocument.ts

```diff
@@ -87,19 +87,19 @@ export class ReadonlyDocument implements vscode.TextDocument {
         return ret;
     }
 
-    initialize(fileContent: string, languageId: string) {
-        this.fileContent = fileContent.split('\r\n').join('\n').split('\n').join('\r\n');//to make sure every line ends with \r\n
+    initialize(rawContent: string, langId: string) {
+        this.fileContent = rawContent.split('\r\n').join('\n').split('\n').join('\r\n');//to make sure every line ends with \r\n
         this.eol = vscode.EndOfLine.CRLF;
         this.eolLength = 2;
 
-        this.languageId = languageId;
+        this.languageId = langId;
 
         if (this.fileContent.length === 0) {
             this.lineCount = 0;
             this.lines = [];
         }
         else {
-            this.lines = fileContent.split('\r\n');
+            this.lines = this.fileContent.split('\r\n');
```

Comment:
I tested this on my mochawesome PR branch by correcting this line. It ran all the tests and opened up in the browser perfectly after implementing your OS specific fix.